.pending/breaking/gaiacli/40-rest-server-s---c

```diff
@@ -0,0 +1 @@
+#40 rest-server's --cors option is now gone.
```

Comment:
Let's remove this entirely. This raises the question of whether or not we should duplicate change log entries (I assume the SDK already has this in its pending log).

Reply:
Well, it's a change in gaiacli `rest-server` interface, isn't it? I think it's worth mentioning the removal of a flag. I suspect I may have gotten wrong the sdk's changelog entry anyway :-/

Reply:
I'll rewrite SDK's pending entry once this is merged https://github.com/cosmos/cosmos-sdk/commit/8cc708b67cf8cf06484bb3571eeead0103f560a8

Reply:
I don't think we should duplicate changelogs...Gaia should ultimately have a pending long entry to an updated SDK version which contains all the updates. Otherwise, we duplicate everything. Also, this PR is much more than just removing a CLI flag.

Reply:
The removal of the `--cors` flag is an ABI change in `gaiacli` and affect usage of the built binary. It generally seems reasonable to mention changes to a program in the changelog it is shipped with.

Reply:
True, but what about once the SDK is released and we have a single PR in Gaia to update the SDK version. This SDK version changes many things (CLI, binaries, APIs, etc...). Do we include all of these in Gaia's pending log too? If so, that means we're gonna have to either:

1. Copy/paste the entire SDK change log over
2. Read change-by-change and cherry-pick

I think (2) is better, but very burdensome. Do you agree?

Reply:
Oh I see now - 2. is suboptimal, whilst 1. is horrible. Let's go for 2. :+1: